codecov_auth/models.py

```diff
@@ -471,6 +475,48 @@ def remove_admin(self, user):
         self.save()
 
 
+GITHUB_APP_INSTALLATION_DEFAULT_NAME = "codecov_app_installation"
+
+
+class GithubAppInstallation(
+    ExportModelOperationsMixin("codecov_auth.github_app_installation"), BaseCodecovModel
+):
+
+    # replacement for owner.integration_id
+    # installation id GitHub sends us in the installation-related webhook events
+    installation_id = models.IntegerField(null=False, blank=False)
+    name = models.TextField(default=GITHUB_APP_INSTALLATION_DEFAULT_NAME)
```

Comment:
what is this for?

Reply:
I think this is what will differentiate the type of apps. As you mentioned above, we might use sentry's app or support multiple apps. This is how we know which is which.